package.json

```diff
@@ -30,11 +31,13 @@
   },
   "homepage": "https://github.com/mcollina/avvio#readme",
   "devDependencies": {
+    "@types/node": "^10.3.0",
     "express": "^4.16.3",
     "pre-commit": "^1.2.2",
     "standard": "^11.0.0",
     "tap": "^11.1.3",
-    "then-sleep": "^1.0.1"
+    "then-sleep": "^1.0.1",
+    "typescript": "^2.9.1"
   },
```

Comment:
No need to reference the types? https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
index.d.ts is the default file for typescript definition

Reply:
Good to know, thanks.